File: src/codility/java/timecomplexity/PermMissingElem.java
package codility.java.timecomplexity;

public class PermMissingElem {
    public int solution(int[] A) {
        double N = A.length +1;

        double sum = (N * (N+1)) /2;

        for (int a : A) {
            sum -= a;
        }
        return (int)sum;
    }
}

File: src/codility/kotlin/PermMissingElem.kt
package codility.kotlin

class PermMissingElem {
    fun solution(A: IntArray): Int {
        // write your code in Kotlin
        val N = (A.size +1).toLong()

        var sum = (N * (N +1))/ 2

        for (a in A) {
            sum -= a.toLong()
        }

        return sum.toInt()
    }

}

